Parser/Device/HbbTv.php

```diff
@@ -38,8 +38,8 @@ class HbbTv extends AbstractDeviceParser
      */
     public function parse(): ?array
     {
-        // only parse user agents containing hbbtv fragment
-        if (null === $this->isHbbTv()) {
+        // only parse user agents containing fragments: hbbtv, shell
+        if (null === $this->isHbbTv() && false === $this->isShellTv()) {
```

Comment:
Wondering if we should rename the class, as it now also handles ShellTV 🤔. Or alternatively we move all the ShellTV detections to a new parser, that only handles the shelltv stuff.

Reply:
I think it's better to make a separate TvParser

Reply:
You can also add an additional `ShellTvParser`. The advantage of having separate parsers is, that they can be used standalone if someone needs it.